apple/internal/experimental_mixed_language_library.bzl

```diff
@@ -288,14 +291,27 @@ target only contains Objective-C files.""")
 
     # Add Obj-C includes to Swift header search paths
     repository_name = native.repository_name()
-    includes = kwargs.get("includes", [])
+    includes = kwargs.pop("includes", [])
     for x in includes:
         include = x if repository_name == "@" else "external/" + repository_name.lstrip("@") + "/" + x
         swift_copts += [
             "-Xcc",
             "-I{}".format(include),
         ]
```

Comment:
It's intentional that this doesn't include the `header_map_ctx.includes`, correct?

Reply:
Yeah its not needed since it's not needed from what I've seen. The headermap include should be propagated to downstream deps via the `objc_library1`